src/src/WebServer/AdvancedConfigPage.cpp

```diff
@@ -191,7 +191,7 @@ void handle_advanced() {
 
 
   addFormSubHeader(F("Inter-ESPEasy Network"));
-
+  if (Settings.UDPPort != 8266 ) addFormNote(F("Preffered P2P port is 8266"));
```

Comment:
Typo, should be `Preferred`

Reply:
And I'm not sure `P2P` is the right term to use in this context, as the group header says `Inter-ESPEasy Network`, and the setting `UDP port`, the only place where P2P is used is controller C013. 😕

Reply:
This UDP port is only used on the ESPEasy p2p layer.
It used to be part of the core of ESPEasy, but I found it more suitable as controller as it would otherwise lead to lot of special cases to handle where as a controller it is clear where it should be applied.

Reply:
N.B. the correct spelling is `preferred`